packages/webchat/src/store/index.ts

```diff
@@ -240,7 +245,8 @@ class RootStore {
 
   @action.bound
   async fetchPreferences(): Promise<void> {
-    const preferences = await this.api.fetchPreferences()
+    // TODO: where to fetch this from? Can this just be set in the frontend?
+    const preferences = { language: 'en' } // await this.api.fetchPreferences()
```

Comment:
This comes from the user's attributes, and it affects the webchat UI and the language in which the bot will respond.  When you enable the bot's details page, you also have a drop down to change language. Just storing the lang in the frontend won't do it